ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1875,6 +1868,30 @@ AP_ServoRelayEvents *GCS_MAVLINK_Copter::get_servorelayevents() const
     return &copter.ServoRelayEvents;
 }
 
+#if ADVANCED_FAILSAFE == ENABLED
+AP_AdvancedFailsafe *GCS_MAVLINK_Copter::get_advanced_failsafe() const
+{
+    return &copter.g2.afs;
+}
+#endif
+
+MAV_RESULT GCS_MAVLINK_Copter::handle_flight_termination(const mavlink_command_long_t &packet) {
+    MAV_RESULT result = MAV_RESULT_FAILED;
+
+#if ADVANCED_FAILSAFE == ENABLED
+    if (super(packet) != MAV_RESULT_ACCEPTED) {
```

Comment:
@WickedShell It doesn't make sense to me that when you have AFS compiled in, you let it decide what do in some cases but not in others - if it was just checking if AFS was enabled, and if not, disarm, I would understand, but not for every time the AFS says no.

At this point I should leave this for @rmackay9 to decide, since he's the Copter maintainer, not me.